modules/admin_manual/pages/configuration/server/excluded_blacklisted_files.adoc

```diff
@@ -21,6 +20,18 @@ file explorer, the path cannot be accessed.
 For example configurations please see
 `owncloud/config/config.sample.php`.
 
+[NOTE]
+====
+Many filesystems are, by default, case-sensitive. 
+This means that multiple files or folders with the same name cannot exist in the same directory.
+Therefore no differentiation is made in processing, as it would just return a deny at a later stage.
```

Comment:
I don't quite follow what is meant in this sentence. Can you clarify what you mean by "no differentiation is made in processing"?

Reply:
blacklist files and excluded directories - as the name says do things for files or dirs. But due to some technical limitations outside owncloud as described, they are treated the same in processing.

Reply:
Your rewrite regarding `case-sensitive`ness is not the point. It is about that a file and directory having the same name CAN NOT coexist. In SMB it is even worse as it does not differenciate between upper and lower case. My text reflected that clearly...

Reply:
I disagree that the change is wrong. It took what you originally wrote and shortened it.

Reply:
I still need clarification on what "as it would just return a deny at a later stage." means, please. As someone new to this content, it's not clear.

Reply:
Imagine that on the oC side, no restriction regarding an exclude is set. But the backend has a hidden directory lets name it `.snapshot`. OC itself passes the create request to the backend but the backend denies it and returns an error. You cant see the hidden dir in oC but you get a deny from the backend for a name you think it is ok. This is later stage. While using exclude, you get an oC triggered deny which is early stage.

Reply:
I appreciate, better, what you're saying. But what is the "deny"? Is it an HTTP status code, does it come in the form of a JSON response, etc?

Reply:
The response sent from the backend is translated by oC to a http status code, usually a 403. But due to the nature of the various backends, this may vary. But this does not matter as either the user or a client gets a response telling „you cannot do this“ (most likely 403). The reason I wrote my text in that way is exactly that everyone understands „deny“ without being specific to a http response.